05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,705 @@
+
+If a dataset had WT0 for the main weight and had 20 BRR weights indicated WT1, WT2, ..., WT20, use the following syntax:
+
+```{r}
+#| label: repgen
+#| eval: FALSE
+des_brr <- dat %>%
+  as_survey_rep(weights = WT0, repweights= all_of(str_c("WT", 1:20)),
+                type="BRR", mse=TRUE)
+
+des_brr <- dat %>%
+  as_survey_rep(weights = WT0, repweights= num_range("WT", 1:20),
+                type="BRR", mse=TRUE)
+
+```
+
+If a dataset had WT for the main weight and had 20 BRR weights indicated REPWT1, WT2, ..., REPWT20, the following syntax could be used (both are valid):
+
+```{r}
+#| label: brr
+#| eval: FALSE
+des_brr <- dat %>%
+  as_survey_rep(weights = WT, repweights= all_of(str_c("REPWT", 1:20)),
+                type="BRR", mse=TRUE)
+
+des_brr <- dat %>%
+  as_survey_rep(weights = WT, repweights= starts_with("REPWT"),
+                type="BRR", mse=TRUE)
+```
+
+If the replicate weight variables are on the file consecutively, the following syntax can also be used:
+
+```{r}
+#| label: brr2
+#| eval: FALSE
+des_brr <- dat %>%
+  as_survey_rep(weights = WT, repweights= REPWT1:REPWT20, type="BRR",
+                mse=TRUE)
+```
+
+Typically, the replicate weights sum to a value similar to the main weight as they are both supposed to provide population estimates. Rarely, an alternative method will be used where the replicate weights have values of 0 or 2 in the case of BRR weights. In this case, the replicate weights are not combined and the option `combined_weights = FALSE` should be indicated with the default value being TRUE. This specific syntax is shown below:
+
+```{r}
+#| label: brr3
+#| eval: FALSE
+des_brr <- dat %>%
+  as_survey_rep(weights = WT, repweights= starts_with("REPWT"), 
+                type="BRR", combined_weights = FALSE, mse=TRUE)
+```
+
+#### Example {-} 
+
+The {survey} package includes a data example from Section 12.2 of Levy and Lemeshow. In this fictional data, two out of five ambulance stations were sampled from each of three emergency service areas (ESAs) thus BRR weights are appropriate with 2 PSUs (stations) sampled in each stratum (ESA). 
+
+```{r}
+#| label: brr_display
+data(scd)
+scdbrr <- scd %>%
+  as_tibble() %>%
+  mutate(
+    wt=5/2,
+    rep1 = 2 * c(1, 0, 1, 0, 1, 0),
+    rep2 = 2 * c(1, 0, 0, 1, 0, 1),
+    rep3 = 2 * c(0, 1, 1, 0, 0, 1),
+    rep4 = 2 * c(0, 1, 0, 1, 1, 0))
+
+scdbrr
+```
+
+In the code above, BRR weights are created as was done in Levy and Lemeshow. To specify the BRR weights, the following syntax is used:
```

Comment:
I would move this first sentence above the code to specify what you're about to do.